src/main/webapp/js/common.js

```diff
@@ -943,7 +945,7 @@ function toggleSingleCollapse(e) {
     if ($(e.target).is('a') || $(e.target).is('input')) {
         return;
     }
-    var glyphIcon = $(this).find('.glyphicon');
+    var glyphIcon = $(window).find('.glyphicon');
```

Comment:
I don't think this is quite right, `$(this)` refers to the element that the event is binded to.